Use only theme colors in tailwind and mui

dmijatovic opened the pull request:
# Use only theme colors

Closes #823

Changes proposed in this pull request:
*  We changed tailwind.config to use only theme colors. The tailwind plugin will suggest all possible color options.    

How to test:
* `make start` to build app
* navigate all pages for unexpected/different color use compared to live/dev version

PR Checklist:

*   [ ] Increase version numbers in `docker-compose.yml`
*   [ ] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

ctwhome commented:
That's correct, lw the background is indeed gray!

On Wed, May 17, 2023, 10:00 Jason Maassen ***@***.***> wrote:

> The background seems to have changed from white to grey:
>
> [image: Screenshot 2023-05-17 at 09-56-57 Organisations Research Software
> Directory]
> <https://user-images.githubusercontent.com/1705533/238875677-34af1924-c8ef-4d97-865e-5dd6ac45106f.png>
>
> [image: Screenshot 2023-05-17 at 09-57-09 Software Research Software
> Directory]
> <https://user-images.githubusercontent.com/1705533/238875725-be0919e5-566b-4df3-898f-47cd0ea8cf3a.png>
>
> I'm using firefox on linux
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/research-software-directory/RSD-as-a-service/pull/849#issuecomment-1550935357>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAAJXRBA4VIWKUKVYSOPVTXGSAQRANCNFSM6AAAAAAXC64H5A>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

dmijatovic commented:
@ctwhome I have updated tailwind.config to extend the default color palette instead of using only theme colors (based on our latest discussion). I have added all tailwind colors used in new v2 pages to theme colors (settings.json). Can you please review?

ctwhome reviewed (approved):
It looks like it was expected. I like the comment on being careful and wise about adding custom colors to the theme settings.
